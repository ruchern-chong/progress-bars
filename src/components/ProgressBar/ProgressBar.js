import React from 'react';
import PropTypes from 'prop-types';

const ProgressBar = props => {
  const { bar } = props;

  return <progress value={bar} max={100} />;
};

ProgressBar.propTypes = {
  bar: PropTypes.number
};

export default ProgressBar;
