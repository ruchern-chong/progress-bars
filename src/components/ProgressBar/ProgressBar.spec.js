import React from 'react';
import { render } from '@testing-library/react';
import ProgressBar from './ProgressBar';

describe('ProgressBar', () => {
  test('should render', () => {
    const { container } = render(<ProgressBar />);

    expect(container).toMatchSnapshot();
  });
});
