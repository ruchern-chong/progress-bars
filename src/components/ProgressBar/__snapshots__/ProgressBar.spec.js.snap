// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar should render 1`] = `
<div>
  <progress
    max="100"
  />
</div>
`;
